app/assets/javascripts/core/gtm.js

```diff
@@ -0,0 +1,56 @@
+(function() {
+
+  function filterFormSubmitListner(event) {
+    window.dataLayer = window.dataLayer || [];
+    var form = document.querySelector('form[name="organisation-picker"]');
+
+    var date_range = form.querySelector('input[name="date_range"]:checked').value;
```

Comment:
Shall we check that these selectors return anything before pushing to dataLayer? as we did with the search term?

Reply:
Fixed in 42758de